.github/workflows/awsfulltest.yml

```
@@ -0,0 +1,31 @@
+name: nf-core AWS full size tests
+# This workflow is triggered on published releases.
+# It can be additionally triggered manually with GitHub actions workflow dispatch button.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+jobs:
+  run-tower:
+    name: Run AWS full tests
+    if: github.repository == 'nf-core/rnasplice'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Launch workflow via tower
+        uses: seqeralabs/action-tower-launch@v1
+        with:
+          workspace_id: ${{ secrets.TOWER_WORKSPACE_ID }}
+          access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}
+          compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
+          workdir: s3://${{ secrets.AWS_S3_BUCKET }}/work/rnasplice/work-${{ github.sha }}
+          parameters: |
+            {
+              "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/rnasplice/results-${{ github.sha }}"
+            }
+          profiles: test_full,aws_tower
```

```suggestion
          profiles: test_full
```
No need for that profile anymore